ghost/core/core/frontend/web/site.js

```diff
@@ -75,6 +75,9 @@ module.exports = function setupSiteApp(routerConfig) {
     // Comment counts
     siteApp.use(mw.servePublicFile('built', 'public/comment-counts.min.js', 'application/javascript', constants.ONE_YEAR_S));
 
+    // Comment counts
```

Comment:
Looks like a copy/paste. This comment doesn't match below.

Reply:
```suggestion
    // Member attribution
```